core/invoker/src/main/scala/whisk/core/container/package.scala

```diff
@@ -83,7 +83,9 @@ package object container {
         override def toString() = s"$host:$port"
     }
 
-    sealed abstract class ContainerIdentifier(val id: String)
```

Comment:
The types are:

1. `Container.containerName` = `Option[ContainerName]`
2. `Container.name` = `ContainerName` ( @psuter  proposes to change to `String`)
3. `ContainerName.name` = `String`